ape_alchemy/provider.py

```diff
@@ -203,6 +203,11 @@ def make_request(self, rpc: str, parameters: Optional[Iterable] = None) -> Any:
             )
             raise cls(message) from err
 
+        if isinstance(result, dict) and (res := result.get("result")):
```

Comment:
bugfix here, does the same as ape-ethereum, thus causing `eth_call` (and other RPCs) to function correctly.

Reply:
@johnson2427 it processes it here (like ape core does) and returns the extracted value

this was kind of a regression because as part of the optimizations in core, switched the base class to use make_request more, causing us to reach this bug